.retest/recheck.ignore

```diff
@@ -0,0 +1,6 @@
+# ignore non-visible attributes
```

Comment:
Both ignore files are simply improvements for our own recheck-web tests. If they work out, we can also think about automatically creating them in case no ignore files can be found.

Reply:
I would give them as examples, but like the .gitignore is not filled with e.g. `*/bin`, I would not fill those, but instead point to examples in the comment.

Reply:
Indeed, good idea!